grakn-core/src/main/java/ai/grakn/util/REST.java

```diff
@@ -42,44 +48,45 @@
         public static class KB {
             @Deprecated
             public static final String GRAQL = "/kb/graql";
-            public static final String ANY_GRAQL = "/kb/graql/execute";
+            public static final String ANY_GRAQL = "/kb/:keyspace/graql";
         }
 
         /**
          * URIs to Tasks Controller endpoints
          */
         public static class Tasks {
```

Comment:
Aren't tasks always linked with a specific keyspace? If that is true shouldn't that keyspace be on the path?

Reply:
There's nothing _requiring_ tasks to have a keyspace param, but I think you might be right that they all do now. We can work on that in another PR, I'm worried about how long this one is becoming!